src/complex/Filter/Output.hpp

```diff
@@ -107,13 +107,36 @@ class COMPLEX_EXPORT IDataCreationAction : public IDataAction
   DataPath m_CreatedPath;
 };
 
+/**
+ * @brief
+ */
+struct COMPLEX_EXPORT DataObjectModification
+{
+
+  enum class ModifiedType : uint64
```

Comment:
Storing `DataObject::Type` would allow to not have to duplicate modified/possibly deleted for each type of object.